---
title: "Analysis Report 2: Your Title Here"
author: "Brian Rezende"
date: "November 21, 2018"
output: github_document
bibliography: references.bib
csl: bioinformatics.csl
---

*Overall, a single-spaced page is about 500 words. So if the guidelines say half of a page, think about writing around 250 words. You can use the wordcountaddin in RStudio to track your progress.*

# Introduction

Add about 1 page here. Must cite at least 5 peer reviewed articles.

# Methods

## Sample origin and sequencing

Add about half a page here. In this section instead of first person (I/we), use Li et al. [@li2015rna] and Seo et al. [@seo2012transcriptional], since you'll just be describing what they did, based on the methods in their paper(s). It's important to include this so the reader knows what the study design was for the data you will be presenting.

## Computational

These are the methods that were used to process the sequencing data. Should probably be at least a half of a page. At a very minimum should include citations for biomartr, trimmomatic, and sailfish. Note that these three methods references don't count towards the five references you need to cite in the introduction.

# Results

In addition to a minimum of 4-5 figures/tables (and associated captions), you should include sufficient text in this section to describe what your findings were. Remember that in the results section you just describe what you found, but you don't interpret it - that happens in the discussion.

```{r load-libraries, message = FALSE, echo = FALSE}
# Be sure to install these packages before running this script
# They can be installed either with the intall.packages() function
# or with the 'Packages' pane in RStudio

# load general-use packages
library("dplyr")
library("tidyr")
library("knitr")
library("ggplot2")
library("magrittr")

# this package allows for the easy inclusion of literature citations in our Rmd
# more info here: https://github.com/crsh/citr
# and here:
# http://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html
library("citr")
```

```{r load-data, message = FALSE, echo = FALSE}
# load the dataset from a compressed binary file
# it gets loaded as an object called "joined_table"
# this has 6.2 million rows...so you will need to be thoughtful about
# how you analyze the data so that you don't overwhelm your laptop
load("output/final_compiled_counts/joined_count_data.RData")

# test that it loaded correctly before proceeding
stopifnot(exists("joined_table"))
```

```{r make-summary-table, echo = FALSE}
# There are many many genes in this dataset, so we can
# subset it down to just a few here to look for interesting patterns
# in the most highly expressed
top_15 <- joined_table %>%
  group_by(gender, genename) %>%
  summarize(mean_count = mean(counts_lengthscaledtpm)) %>%
  arrange(desc(mean_count)) %>%
  head(n = 15)


# then we can use the `kable()` function to make a nicely formatted
# markdown table
top_15 %>%
  kable()
```

**Table 1**: The most highly expressed genes in both genders included *SFTPB* and *EEF1A1*.

```{r make-barplot-of-highly-expressed-genes, echo = FALSE}
# this code uses the same data as above, but use it to make a
# barplot - remember geom_col() is just like
# when you use geom_bar(stat = "identity")
top_15 %>%
  ggplot(aes(x = genename,
             y = mean_count,
             fill = gender)) +
    geom_col(position = "dodge")
```

**Figure 1**: Here we show an example figure caption.

```{r make-boxplot-of-highly-expressed-genes, echo = FALSE}
# here we just want to pull out the unique gene names and turn
# them into a vector so we can use it below to make a boxplot
# we use the pull() funtion to get this as a vector, just like
# we did when making histograms several weeks ago
top_genes <- top_15 %>%
  ungroup() %>%
  select(genename) %>%
  unique() %>%
  pull()

# now we need to filter from the full data set again, because
# we don't just want summary data, we want all the data in
# order to make boxplots
joined_table %>%
  filter(genename %in% top_genes) %>%
  ggplot(aes(x = genename,
             y = counts_lengthscaledtpm,
             fill = gender)) +
    geom_boxplot() +
    facet_wrap(~cancer_stage) +
    xlab("Gene Name") +
    ylab("Scaled read counts per gene") +
    ggtitle("Read counts per gene by gender and cancer stage") +
    theme_bw() + # simplifies theme
    theme(axis.text.x = # rotates x axis labels vertically
            element_text(angle = 90,
                         hjust = 1))
```

**Figure 2**: Here we show another example figure caption.

```{r KRAS gene presence in diff ages, gender, and smoking status}
# look at trend of smoking status w/ age and gender 
joined_table %>%
  filter(genename == "KRAS")%>%
ggplot(aes(x = age_at_diagnosis,
           y = counts_lengthscaledtpm,
           color = gender,
           shape = smoking_status))+
  geom_point() +
  xlab("Age at Diagnosis") +
  ylab("Scaled read counts per gene") +
  ggtitle("Read counts of KRAS in different ages")
  theme_bw() +
  theme(axis.text.x = element_text(angle = 90,
                         hjust = 1))
```
**Figure 1:** Occurrence of the KRAS gene in male and female patients with 
varying stages of cancer and ages of diagnosis. There is an increased distribu
tion in ages ranging from 58-75. 
```{r bargraph-of-EGFR-and-KRAS-presence-in-tissues}
# EGFR and KRAS were found to be mutually exclusive in the literature
# so I wanted to compare their expression in cancer tissue vs normal tissues
top_2 <- joined_table %>%
  filter(genename %in%
  c("KRAS","EGFR"))
  
top_2 %>%
  ggplot(aes(x = genename,
             y = counts_lengthscaledtpm,
             fill = normal_or_cancer))+
  geom_col(position = position_dodge()) +
  xlab("Gene Name") +
  ylab("Scaled read counts per gene") +
  ggtitle("KRAS & EGFR Tissue Analysis")
```
**Figure 2:** Prevalence of mutually exlusive genes, EGFR and KRAS, in normal
and cancer tissue samples. The prevalence was measured by the read counts per
gene. It accounts for both sexes and all ages. KRAS greatly overshadows the 
presence of EGFR in both cancer and normal tissue samples. 
```{r boxplot-of-mutually exclusive genes}
top_2 %>%
   ggplot(aes(x = smoking_status,
             y = counts_lengthscaledtpm,
             fill = genename )) +
   geom_boxplot() +
    facet_wrap(~cancer_stage) +
    xlab("Smoking Status") +
    ylab("Scaled read counts per gene") +
    ggtitle("Read counts of KRAS & EGRF by cancer stage and status") 
    theme_bw()
```
**Figure 3:** Smoking status compared to the read counts per gene of the two
mutually exclusive genes. Each stage is taken into account to track development
and increased prevalence. 
```{r point-graph-of-KRAS-presence-in-cancer-stages}
kras_subset <- joined_table %>%
  filter(genename %in%
  c("KRAS"))

kras_subset %>%
  ggplot(aes(x = cancer_stage,
             y = age_at_diagnosis,
             color = genename))+
  geom_point()+
  xlab("Cancer Stage") +
  ylab("Age at Diagnosis") +
  ggtitle("KRAS Presence in Cancer Stages and Ages")

```
**Figure 4:** Inferred presence of KRAS in different ages of diagnosis and the
corresponding cancer stages. Possible preventative treatments may also be 
inferred. KRAS demonstrates an abundance in early stages and in older 
populations. 

# Discussion

Add around 1-2 pages interpreting your results and considering future directions one might take in analyzing these data.

# Sources Cited
